### PR TITLE
Update sha256 labels

### PR DESCRIPTION
**Pull request by ruffsl**
See https://github.com/opencontainers/image-spec/blob/v1.0.1/annotations.md#rules

Related:
* https://github.com/docker-library/official-images/pull/8786#issuecomment-700238992
* https://github.com/osrf/docker_templates/pull/92

### CODE REVIEW

**Comment by ruffsl** [comment]
Not sure when patch in e968311 was removed, nor if it's still needed, but I just added it back, as the image template still hadn't removed it.

File: ros/eloquent/ubuntu/bionic/images.yaml.em [hunk 1]
```diff
@@ -43,10 +43,3 @@ images:
             - ros-comm
             - roscpp-tutorials
             - rospy-tutorials
-        downstream_packages:
```
Comment (mikaelarguedas) [inline ros/eloquent/ubuntu/bionic/images.yaml.em:46]: Is this not necessary anymore ?

File: ros/eloquent/ubuntu/bionic/ros1-bridge/Dockerfile [hunk 2]
```diff
@@ -35,11 +35,6 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     ros-eloquent-demo-nodes-py=0.8.4-1* \
     && rm -rf /var/lib/apt/lists/*
 
-# install downstream packages
```
Comment (mikaelarguedas) [inline ros/eloquent/ubuntu/bionic/ros1-bridge/Dockerfile:38]: Is this not necessary anymore ?

**Review by mikaelarguedas — COMMENTED** [review]
> Not sure when patch in e968311 was removed, nor if it's still needed, but I just added it back, as the image template still hadn't removed it.

Looks like you added it back only for dashing but that you removed it for eloquent. Is it on purpose ?

File: ros/eloquent/ubuntu/bionic/images.yaml.em (hunk 1 above)
Comment (ruffsl) [reply ros/eloquent/ubuntu/bionic/images.yaml.em:46]: I looked at the ros_ci.yaml to remember to check which distro's where patched, but only saw noetic being treated as special. Patch re-applied here: https://github.com/osrf/docker_images/pull/479

File: ros/eloquent/ubuntu/bionic/ros1-bridge/Dockerfile (hunk 2 above)
Comment (ruffsl) [reply ros/eloquent/ubuntu/bionic/ros1-bridge/Dockerfile:38]: https://github.com/osrf/docker_images/pull/478#discussion_r506654208

**Pull request merged**